packages/now-cli/src/util/dev/builder-cache.ts

```diff
@@ -31,12 +31,15 @@ declare const __non_webpack_require__: typeof require;
 
 const registryTypes = new Set(['version', 'tag', 'range']);
 
+const nowStaticBuilder: BuilderWithPackage = {
+  runInProcess: true,
+  builder: Object.freeze(staticBuilder),
+  package: Object.freeze({ name: '@now/static', version: '' }),
```

Comment:
@TooTallNate I think this is a bug because `@now/static` is used for both vercel and now scopes below.

Reply:
1b88cd98d8e839ce8749607469a60ca32c9d4b8d